frontend_tests/zjsunit/index.js

```diff
@@ -61,7 +61,7 @@ Module.prototype.hot = {
 function short_tb(tb) {
     const lines = tb.split("\n");
 
-    const i = lines.findIndex((line) => line.includes("Module._compile"));
+    const i = lines.map((line) => line.includes("Module._compile")).lastIndexOf(true);
```

Comment:
Can you extend the commit message with an example of what the impact is?  This seems right to me logically but it'd be a lot clearer with an example.

Reply:
I’ll extract this to a separate PR, because now I’m not really sure what the intention was with 5bd73ce1909b46078b9576101be250fda639a826 (#17367).